clients/gtest/gemm_ex_gtest.cpp

```diff
@@ -291,8 +291,7 @@ class parameterized_gemm_ex : public ::TestWithParam<gemm_ex_tuple>
 TEST(known_bugs_int8, IdentityMatrix)
 {
     // Simple test cases running on matrices full of ones
```

Comment:
`rocblas_local_handle` is like `rocblas_handle` except that it is automatically created and destroyed.